com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Artistic/Mask/ColorMaskNode.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class ColorMaskNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,     // Version
+            "ColorMask", // Name
+@"
+{
+    Distance = distance(MaskColor, In);
+    Out = saturate(1 - (Distance - Range) / max(Fuzziness, 1e-5));
```

Comment:
Maybe we can get rid of Distance since it is only used once?

Reply:
Done!  Thanks for the tip!